common/hashes/jarm/jarmhash.go

```
@@ -49,21 +49,26 @@ func fingerprint(dialer *fastdialer.Dialer, t target, duration int) string {
-		_ = conn.SetReadDeadline(time.Now().Add(timeout))
-		buff := make([]byte, 1484)
-		_, _ = conn.Read(buff)
-		_ = conn.Close()
-		ans, err := jarm.ParseServerHello(buff, probe)
-		if err != nil {
-			return ""
-		}
-		results = append(results, ans)
+
+		func() {
+			defer conn.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `conn.Close` is not checked

(errcheck)